README.md

```diff
@@ -10,3 +10,8 @@ This plugin is for the [Intake data loader library](http://intake.readthedocs.io
 ```
 conda install -c intake intake-accumulo
 ```
+
+### Caveats
+
+This plugin has been verified with Accumulo 1.5.2, pyaccumulo 1.5.x, and Python
+2.7. Support for other configurations is unverified at this time.
```

Comment:
Still true, even though we test for py3 and build noarch?

Reply:
Actually, we don't depend on pyaccumulo at all now, so this must be old. Do we depend on thrift?